README.md

```diff
@@ -27,31 +29,109 @@ npm install @dojo/widgets
+To use a widget in your application, you will need to import each widget individually:
+
+```ts
+import Button from '@dojo/widgets/button/Button';
+```
+
+Each widget module has a default export of the widget itself, as well as named exports for things such as properties specific to the widget:
+
+```ts
+import Button, { ButtonProperties } from '@dojo/widgets/button/Button';
+```
+
+Because each widget is a separate module, when you create a release build of your application, you will only include the widgets that you have explicitly imported. 
 
 ## Features
 
-Dojo 2 widgets are supported in all evergreen browsers (IE11+, Chrome, Firefox, Safari) as well as popular mobile browsers (Mobile Safari, Chrome on Android).
+- All widgets are supported in all evergreen browsers (IE11+, Chrome, Firefox, Safari) as well as popular mobile browsers (Mobile Safari, Chrome on Android).
+
+- All widgets support accessibility (`a11y`) out of the box.
+
+- All widgets are fully themeable.
+In a future release we will ship with several official Dojo 2 themes that can be applied to the widgets (all themes will live in a separate repo).
+
+- All widgets support internationalisation (`i18n`)
+
+## Widgets
+
+### Form widgets
+[Button](src/button/README.md)
+
+[Checkbox/Switch](src/checkbox/README.md)
+
+[ComboBox](src/combobox/README.md)
+
+[Label](src/label/README.md)
+
+[Radio](src/radio/README.md)
+
+[Select](src/select/README.md)
+
+[Slider](src/slider/README.md)
 
-Dojo 2 widgets are fully accessible (`a11y`), out of the box.
+[TextArea](src/TextArea/README.md)
 
-Dojo 2 widgets are fully themeable. In a future release we will ship with several official Dojo 2 themes that can be applied to the widgets (all themes will live in a separate repo).
+[TextInput](src/TextInput/README.md)
 
-Dojo 2 widgets are fully internationalisable (`i18n`)
+### Layout widgets
+[SlidePane](src/slidepane/README.md)
+
+[TabPane](src/tabpane/README.md)
+
+### Misc widgets
+[Dialog](src/dialog/README.md)
 
 ## Conventions
 
+### EventHandlers
+
+You can register event handlers that get called when the corresponding events occur, by passing the handlers into a widget, in their `properties`.
+The naming convention for event handlers is as follows:
+
+- if the parent of the widget has the power to decide *if* an event is successful, i.e. can cancel the event, then the child widget will call an event handler in the following format:
+
+`onRequest[X]`, e.g. for a `close` event, the event handler called by the child widget must be called `onRequestClose`
+
+- for events that will occur regardless of child/parent interaction, then the `Request` naming convention is dropped:
+
+`on[X]`, e.g. for a `dismiss` event, then event handler called by the child widget must be called `onDismiss`
+
+### Controlled vs uncontrolled form widgets
+Currently, all of our *form* widgets are 'controlled' widgets. This means that a parent widget is responsible to the 'state' of the child widget.
```

Comment:
I'd remove the single quotes around controlled and state... if anything we can define controlled, uncontrolled, state and stateless briefly and then just use the terms